packages/instantsearch.js/src/widgets/hits/__tests__/hits-test.ts

```diff
@@ -82,170 +82,172 @@ describe('hits()', () => {
 
     expect(render).toHaveBeenCalledTimes(2);
     expect(firstRender.props).toMatchInlineSnapshot(`
-{
```

Comment:
this diff is a little weird because it's properly indented now, but the only changes are adding `sendEventToInsights` (and changing `sendEvent` reference, but that doesn't show in the snapshot)